src/NET/MvvmDialogs/DialogTypeLocators/NamingConventionDialogTypeLocator.cs

```diff
@@ -32,14 +33,20 @@ public Type Locate(INotifyPropertyChanged viewModel)
             Type viewModelType = viewModel.GetType();
 
             Type dialogType = Cache.Get(viewModelType);
+
             if (dialogType != null)
             {
                 return dialogType;
             }
 
             string dialogName = GetDialogName(viewModelType);
-            
+
             dialogType = GetAssemblyFromType(viewModelType).GetType(dialogName);
+
+            // if view name ends with "View"
+            if(dialogType == null)
```

Comment:
I think i have a better idea. According to me the whole idea behind this class is to find/search the appropriate View associated with the Provided ViewModel. Right??

1. I think we should re-write the **Locate** Method in such a manner that it should search or locate **View** inside whole project following only one naming convention that **all views must suffix** with **View or view**.

2. Second feature we can provide is that we can write one more **ShowDialog** method so that we can pass other parameters along with ViewModel.

What do you think ?